src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -502,6 +462,13 @@
         // getActives grabs allFields, returns those isActive, sorted.
         let activeFields = this.getActives(component);
 
+        activeFields.forEach(function(currentField) {
+            if (currentField.defaultValue && currentField.type == 'reference') {
+                // lookups in recordeditform store as an array of IDs; need to flatten
```

Comment:
Do we know why they are stored in an array? A lookup would only ever point to a single Id.... right?

Reply:
According to the LWC side of the docs, they allude to the possibility of multiple lookups someday. We're guessing it was easier for them to always assume an array in their code?

Reply:
There is a multi-lookup design pattern in core and in SLDS, so yeah, as Randi said, it looks like they're just working ahead toward a base component for that.

Reply:
Coolbeans. Carry on!